src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js

```diff
@@ -125,4 +124,24 @@ function BaseReportActionContextMenu(props) {
 BaseReportActionContextMenu.propTypes = propTypes;
 BaseReportActionContextMenu.defaultProps = defaultProps;
 
-export default compose(withLocalize, withBetas(), withWindowDimensions)(BaseReportActionContextMenu);
+export default compose(
+    withLocalize,
+    withBetas(),
+    withWindowDimensions,
+    withOnyx({
+        reportActions: {
+            key: ({originalReportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${originalReportID}`,
+            canEvict: false,
+        },
+    }),
+)(
+    memo(BaseReportActionContextMenu, (prevProps, nextProps) => {
```

Comment:
Can you please add code comments to explain why this memo is here and what it's purpose is?